### PR TITLE
Clean up samples to work with the current tooling

### PR DESCRIPTION
**Pull request by Tratcher**
This gives a clean F5 experience in VS.

### CODE REVIEW

**Comment by msotclas** [comment]
Hi __@Tratcher__, I'm your friendly neighborhood Microsoft Open Technologies, Inc. Pull Request Bot (You can call me MSOTBOT). Thanks for your contribution!
    <p>
        It looks like you're a Microsoft contributor (Chris Ross (WEB TOOLS)). If you're full-time, we DON'T require a Contribution License Agreement. If you are a vendor, please DO sign the electronic Contribution License Agreement. It will take 2 minutes and there's no faxing! https://cla2.msopentech.com.
    </p>

TTYL, MSOTBOT;

**Comment by Tratcher** [comment]
@muratg

File: samples/ConsoleApp/project.json [hunk 1]
```diff
@@ -2,6 +2,9 @@
     "dependencies": {
 
     },
+    "commands": {
+        "sample": "ConsoleApp"
```
Comment (davidfowl) [inline samples/ConsoleApp/project.json]: Should be "ConsoleApp": ""ConsoleApp"

**Comment by Tratcher** [comment]
Updated.

**Comment by muratg** [comment]
:shipit:

**Pull request merged**